### ISSUE TITLE
SDK: Failed to send the event to Sentry. Reason: "Bad Gateway".

### ISSUE THREAD

**Issue by Lanser0614** [getsentry/self-hosted#1416]
### Version

2.12

### Steps to Reproduce

Error with conection to laravel

### Expected Result

I can not connect my laravel aplication to Sentry. I am doing all like documentation but it is does not work. 

### Actual Result

![photo_2022-04-07_11-15-23](https://user-images.githubusercontent.com/62260034/162132493-73ff9264-76fe-4dec-9611-57b2fbd2d673.jpg)

**Comment by ljubo2015** [comment]
I have the same issue, after updating on premise sentry to latest version. Before this it was working totally fine.

**Comment by chadwhitacre** [comment]
Still gonna need a lot more info to debug. "Bad Gateway" makes it sound like your server is down or your nginx is misconfigured or your network topology is mixed up. What hypotheses have you tested so far?

**Comment by deutschmannstefan** [comment]
> I have the same issue, after updating on premise sentry to latest version. Before this it was working totally fine.

Same here. Had to downgrade.

**Comment by chadwhitacre** [comment]
@ljubo2015 @deutschmannstefan Are you also using the Laravel SDK?

**Comment by deutschmannstefan** [comment]
@chadwhitacre No. It seems to be a general problem. We send Events from different system (E.g PHP, Swift, Java). No events are received (or even displayed) on the dashboard. From the client site, there seems to be no problem. (I tested one swift implementation right now). After I downgraded the sentry instance, the events come back again.

**Comment by chadwhitacre** [comment]
Gosh. That sounds bad.

So you don't even have the "Bad Gateway" error that @Lanser0614 reports?

**Comment by deutschmannstefan** [comment]
@chadwhitacre No. Seems to be another thing.

**Comment by chadwhitacre** [comment]
@deutschmannstefan Hrm, okay. The next release is going out tomorrow, can you try that and if it still doesn't work open a new ticket?

**Comment by aminvakil** [comment]
https://github.com/getsentry/self-hosted/releases/tag/22.4.0 has been released, can you test it using this release?

**Comment by deutschmannstefan** [comment]
@aminvakil @chadwhitacre Works now with 22.4.0 Thx
